backend/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "dependencies": {
+    "caketype": "^0.5.0",
```

Comment:
With caketype in this repo, I'm concerned that it's a small, non-TSE project maintained by one person (so it may be difficult to fix bugs/make changes in the future) and that it might be confusing for inexperienced developers when we add that layer of complexity on top of the actual onboarding content. Since caketype is not a TSE standard (and I currently don't believe it should be, for the same reasons), IMO we should focus on the core concepts and allow EMs to decide whether to use it in their projects

Reply:
I understand your concern and I agree that it shouldn't be made a standard (EM's should 100% be deciding whatever they're going to use). However, I don't think its inclusion in this project is particularly problematic. We likely will not have very much data validation, the point is more to offer a primer. Caketype's documentation is extremely thorough and I'd argue it does just as good a job at introducing people to the subject as other packages do--if not a better job. 

The tech in this repo is meant to be relatively language/package agnostic meaning the principles can be applied to other projects whether they're on MERN or not. To me, Caketype is a viable solution for our very simple needs and has the added bonus of being TSE-made. If the implementation ends up being arduous or confusing then we can reconsider.

Reply:
But because our needs are so simple here as you say, using caketype or any other special package for handling types seems superfluous. Caketype isn't a data validator either, since you mention that subject--while it supports basic number range, string length, and regex matchers, it currently can't replace something like validator.js/express-validator, which are more standard and better supported libraries and are actually intended for data validation. Caketype itself is primarily a stricter type checker that throws runtime errors if a type doesn't match. Incidentally, validator.js/express-validator also check types/formats (isBoolean, isInt, isISO8601, etc.) and handle errors in a more purpose-built way (e.g. automatically returning responses with error messages).

Nor is caketype TSE-made either, because it's not under the TritonSE organization (would you consider my https://coursegrapher.app to be TSE-made?). I personally do trust that it would be a very useful library to use in any project, inside or outside of TSE, but I don't think future board members should have to rely on our trust and on the hope that Justin will always be available to maintain it.

If we want this repo to be package-agnostic (which I also agree with), then I think it should be limited to those large, commonly used, reliably updated libraries. When caketype reaches 1000 downloads per month (still only 0.3% that of express-validator), then I think it could be taken into consideration.

Reply:
Discussion tabled until next meeting